engine/CMakeLists.txt

```diff
@@ -602,19 +602,6 @@ IF (DATADIR)
     SET(DEFINES "${DEFINES} -DDATA_DIR='\"${DATADIR}\"'")
 ENDIF (DATADIR)
 
-# Gather info about Linux distro and release (if applicable) for later use down below.
```

Comment:
The `FIND_PACKAGE(LinuxDistro)` line needs to go here, not down below. Otherwise, locating python3 fails on Xenial and stretch. (Looks like that's what caused the CI builds to fail.)

Reply:
Interesting since it built locally (though not on that Ubuntu version, since I'm on 21.04 IIRC)

Reply:
Yeah, it's only those particular distro versions that it fails on. At least to my knowledge.

Reply:
Updated against master, which removes Xenial. Let's see if this is still and issue after that.

Reply:
Looks like it's no longer an issue. Cool.